app/uk/gov/nationalarchives/omega/editorial/models/RelatedMaterial.scala

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2022 The National Archives
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of
+ * this software and associated documentation files (the "Software"), to deal in
+ * the Software without restriction, including without limitation the rights to
+ * use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of
+ * the Software, and to permit persons to whom the Software is furnished to do so,
+ * subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+ * FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+ * COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+ * IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+ * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+package uk.gov.nationalarchives.omega.editorial.models
+
+import play.api.libs.json._
+
+case class RelatedMaterial(linkHref: String, linkText: String, description: Option[String] = None)
```

Comment:
Note that the link itself is not optional. This is my interpretation of the [ticket](https://national-archives.atlassian.net/browse/PACT-609) but does not match the prototype:

![Screenshot 2023-01-06 at 12 56 10](https://user-images.githubusercontent.com/10521622/211016708-38031144-f249-4de3-a6db-c6f42276ab09.png)

(my prototype is pink so I can differentiate at a glance)